Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -506,9 +508,23 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
 
     override func applyTheme() {
         super.applyTheme()
-
         reloadData()
     }
+
+    func getAttributedBoldSearchSuggestions(searchPhrase: String, query: String) -> NSAttributedString {
```

Comment:
I saw other places and all classes mostly kept their attributed strings in their own class so decided to keep this method in same class but happy to move it elsewhere